guides/16_Video_Layout.md

```diff
@@ -0,0 +1,527 @@
+
+Typically, there are four steps involved in the lifecycle of a remote video with respect to a single receiving attendee:
+
+1. As remote attendees change their video stream, an up to date list of video sources (ignoring the receiving attendees’ own local video) is provided via the `remoteVideoSourcesDidChange` callback.
+2. The receiving client will select the remote video sources (and possibly which simulcast layer) that they wish to receive, which will initiate a negotiation (or “subscription”) with the backend servers to start forwarding those videos, depending on a client’s downlink policy (explained in a later section).
+
+   - Note that some downlink policies may expose APIs to allow dynamic control over this decision.
+
+3. For each subscribed or unsubscribed video, the receiving attendee will receive a `videoTileDidUpdate` or `videoTileWasRemoved` callback.
+4. To display the video, the receiving attendee can bind or unbind a video tile to a  `<video>` element in their canvas via the [`bindVideoElement`](https://aws.github.io/amazon-chime-sdk-js/interfaces/audiovideofacade.html#bindvideoelement) API.
+
+### Video Uplink and Downlink Policies
+
+Uplink and downlink policies help builders configure the video strategies for sending and receiving video streams in a meeting. The Amazon Chime SDK for JavaScript provides different policies, each with its own configuration and dynamic controls. Builders can choose between the provided policies or extend and create their own, depending on the requirements.
+
+#### Uplink Policy
+
+The video encoding parameters are determined by the uplink policy, which takes into consideration the number of video publishers on the call. If necessary, it may suggest encoding multiple “simulcast” layers (with different bitrates and resolutions). The JavaScript SDK currently provides two uplink policies:
+
+**NScaleVideoUplinkBandwidthPolicy (Default):** In this policy, an attendee encodes and sends one video layer upstream and scales the resolution and bitrate based on the number of video publishers. This is to ensure the receiving attendees who are subscribed to these videos do not experience excessive network or compute load, which may degrade the meeting quality.
+
+**SimulcastUplinkPolicy:** Each attendee may encode and send up to two layers of video with different bitrates and resolutions under this policy. It enables the recipient to switch between the two video layers to adapt to changing network conditions or different layouts (e.g., a featured video may desire a higher resolution than a gallery view video).
+
+- Note: the `SimulcastUplinkPolicy` is currently only supported in Chrome 76 and above versions.
+
+#### Downlink Policy
+
+The downlink policy controls which remote video sources are received downstream based on the network condition, local resources (e.g., CPU), and other factors such as layout changes in the application. The JavaScript SDK currently provides two downlink policies:
```

Comment:
To clarify, are you trying to say: The downlink policy controls which remote video sources are received **as** downstream based on the network condition?